packages/react-scripts-plugin-typescript/src/index.js

```diff
@@ -0,0 +1,35 @@
+'use strict';
+
+const {
+  pushExtensions,
+  pushExclusiveLoader,
+} = require('react-dev-utils/plugins');
+
+function apply(config, { path, paths }) {
```

Comment:
When inlining the plugin during eject the transform is not smart enough to pull required dependencies/packages; so for now required packages must be passed in.